parser/model/ddl.go

```diff
@@ -211,7 +211,13 @@ type DDLReorgMeta struct {
 	SQLMode       mysql.SQLMode                    `json:"sql_mode"`
 	Warnings      map[errors.ErrorID]*terror.Error `json:"warnings"`
 	WarningsCount map[errors.ErrorID]int64         `json:"warnings_count"`
-	Location      *time.Location                   `json:"time_location"`
+	Location      *TimeZone                        `json:"time_zone"`
```

Comment:
Would it break the compatibility?

Reply:
No, I looked at the original PR that added this field that is not on the branch yet.

Reply:
Which PR?

Reply:
PR：https://github.com/pingcap/tidb/pull/29664